components/brave_new_tab_ui/storage/new_tab_storage.ts

```diff
@@ -59,7 +59,7 @@ export const defaultState: NewTab.State = {
   braveTalkPromptAutoDismissed: false,
   braveTalkPromptAllowed: loadTimeData.getBoolean('braveTalkPromptAllowed'),
   rewardsState: {
-    adsAccountStatement: {
+    adsStatementOfAccounts: {
```

Comment:
This may cause trouble for upgrades, where the page state is stored in local storage with the old name and not this new name. Without looking deeper, I can't tell whether that will fail gracefully or not. I would recommend:

* Testing this change out thoroughly for upgrade cases, or
* Forego changing this property name for now.